libmozevent/mercurial.py

```diff
@@ -34,6 +34,9 @@
 MAX_PUSH_RETRIES = 4
 # Wait successive exponential delays: 6sec, 36sec, 3.6min, 21.6min
 PUSH_RETRY_EXPONENTIAL_DELAY = 6
+# Time after which a Phabricator revision should be considered as expired and the
+# try push should no longer be retried
+DIFF_EXPIRY = timedelta(hours=1)
```

Comment:
Let's make this configurable (that is, as a parameter to the MercurialWorker class), so we can add it to the code-review secrets and configure it dynamically.

Reply:
And let's use 24 hours by default instead of 1 (we only want to do this when we're really late).